### PR TITLE
Update dependencies to be D9 compatible

### PR DESCRIPTION
**Pull request by audave**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
     "require": {
         "php": ">=7.0",
         "composer-plugin-api": "^1.1",
-        "composer/semver": "^1.4",
-        "drupal-composer/drupal-scaffold": "^2.2",
+        "composer/semver": "^3",
+        "drupal/core-composer-scaffold": "^9",
```
Comment (pingers) [inline composer.json:10]: This works a bit differently and we'd have to make changes to Handler.php afaik.

Comment (pingers) [reply composer.json:10]: See https://github.com/drupal/core-composer-scaffold/tree/9.1.x vs https://github.com/drupal-composer/drupal-scaffold

Comment (singularo) [reply composer.json:10]: Yeah, I re-did the handler stuff for 9.x - https://github.com/singularo/shepherd-drupal-scaffold/tree/develop/src - probably too different to merge back in, but might help.

**Comment by pingers** [comment]
Superseded by #101

**Pull request closed without merging**